Sources/PayPal/PayPal.swift

```diff
@@ -1,4 +1,4 @@
-#if !COCOAPODS
+#if canImport(PaymentsCore)
```

Comment:
Oh yeh - I also had to make this change here https://github.com/paypal/iOS-SDK/pull/3. I think we need one more ✔️ 👀  on that one!

Reply:
I'll take a look at that PR today!

Reply:
That would be awesome - thank you!